src/java.base/share/classes/sun/security/util/SignatureUtil.java

```diff
@@ -47,32 +47,40 @@
 public class SignatureUtil {
 
     /**
-     * Convert OID.1.2.3.4 or 1.2.3.4 to its matching stdName.
+     * Convert OID.1.2.3.4 or 1.2.3.4 to its matching stdName, and literal
+     * name to upper case.
      *
      * @param algName input, could be in any form
      * @return the matching stdName, or {@code algName} if it is not in the
```

Comment:
nit: simplify this return description and emphasize upper case here also? E.g. the matching stdName or the OID string in upper case.

Reply:
Good point!  I will make this update in this PR.